src/System.Runtime.Serialization.Formatters/pkg/System.Runtime.Serialization.Formatters.pkgproj

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>  
+    <ProjectReference Include="..\ref\System.Runtime.Serialization.Formatters.csproj">
+      <SupportedFramework>net462;netcoreapp1.0</SupportedFramework>
```

Comment:
You don't need to list netstandard1.3, that will be derived from the set of frameworks you explicitly support.  Let's use net46;netcore50;netcoreapp1.0 for now.